Web3/Classes/Core/Web3/Web3.swift

```diff
@@ -281,17 +288,50 @@ public struct Web3 {
             properties.provider.send(request: req, response: response)
         }
         
+        public func sign(
```

Comment:
I am not yet sure about adding private methods to our API. Users can always add them as an extension anyways but I will think about it again...

Reply:
sign is an Ethereum RPC method. It's public one https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_sign

Reply:
Agreed. eth_sign is a standard public method. It's questionable to support personal_sign because it was designed to be used only with a local node, and the 3rd param is a password, although many dapps use them interchangeably due to some bugs / limitations with certain dapp browsers.